applications_tests/lethe-fluid-matrix-free/cylinder_kelly_estimator.prm

```diff
@@ -114,6 +114,16 @@ subsection boundary conditions
   end
 end
 
+#---------------------------------------------------
+# Source term
+#---------------------------------------------------
+
+subsection source term
+  subsection xyz
```

Comment:
What is `subsection xyz` standing for?

Reply:
It is actually the subsection of the source term for the navier-stokes. Not a great name, which is why, I will rename it in a follow PR, because it changes more than 100 files (tests, examples, documentation).

Reply:
The name sucks
It's my mistake from 5 years ago
I take the blame for that